functional_testing/Grid_API_Testing/api_testing/testcases/basic_tests/test09_gws_apis.py

```diff
@@ -415,28 +429,255 @@ def test014_create_gateway_httpproxy(self):
         pass
 
     def test015_create_gateway_portforwards(self):
-        """ GAT-xxx
+        """ GAT-121
         **Test Scenario:**
 
         #. Get random node (N0), should succeed.
+        #. Create bridge(B0) , should succeed.
         #. Create gateway with bridge and vlan as nics should succeed.
         #. Set a portforward form srcip:80 to destination:80
         #. Create one container as a destination host
         #. Start any service in this container
         #. Using core0_client try to request this service and make sure that u can reach the container
+
         """
-        pass
+
+        self.lg.info(" Create bridge(B0) with true nat, should succeed. ")
+        B0_name = self.rand_str()
+        B0_ip = "190.192.4.1"
```

Comment:
if u dont use this variable again, set it inside the config json to make it easy to maintain it